treeherder/webapp/api/runnable_jobs.py

```diff
@@ -57,6 +78,31 @@ def list(self, request, project):
                 'state': 'runnable',
                 'result': 'runnable'})
 
+        for node in tc_graph['tasks']:
+            ret.append({
+                'build_platform_id': datum.build_platform.id,
+                'build_platform': datum.build_platform.platform,
+                'build_os': datum.build_platform.os_name,
+                'build_architecture': datum.build_platform.architecture,
+                'machine_platform_id': datum.machine_platform.id,
+                'platform': node['task']['treeherder']['build']['platform'],
+                'machine_platform_os': datum.machine_platform.os_name,
+                'machine_platform_architecture': datum.machine_platform.architecture,
+                'job_group_id': datum.job_type.job_group.id,
+                'job_group_name': datum.job_type.job_group.name,
+                'job_group_symbol': datum.job_type.job_group.symbol,
+                'job_group_description': datum.job_type.job_group.description,
+                'job_type_id': datum.job_type.id,
+                'job_type_name': datum.job_type.name,
+                'job_type_symbol': datum.job_type.symbol,
+                'job_type_description': datum.job_type.description,
+                'option_collection_hash': datum.option_collection_hash,
+                'ref_data_name': datum.ref_data_name,
```

Comment:
@wlach , this is an example file [https://public-artifacts.taskcluster.net/Zo4UsXv_QH2VdJhSODwwWQ/0/public/all_tasks.json](https://public-artifacts.taskcluster.net/Zo4UsXv_QH2VdJhSODwwWQ/0/public/all_tasks.json)

Yes I've copy pasted it, except for the `build_system_type` field and `platform` field, the easiest to figure out